src/UriTemplateTests/UriTemplateConverterTests.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.ComponentModel;
+using Tavis.UriTemplates;
+using Xunit;
+
+namespace UriTemplateTests
+{
+    public class UriTemplateConverterTests
+    {
+        [Theory]
+        [InlineData("http://example.org/{tenant}/customers")]
```

Comment:
These examples were grabbed from other tests. They seemed as good as any for this test.